ConnectApiHelper.cls

```diff
@@ -170,6 +172,13 @@ global class ConnectApiHelper {
                     messageSegmentInputs.add(inlineImageSegmentInput);
                     strPos = globalMatcher.end();
                 }
+                else if (innerMatchedText.startsWith('record:')) {
+                    // Inline record
+                    String[] recordInfo = innerMatchedText.split(':', 6);
```

Comment:
I don't think `6` is the correct value to use here. According to the [Apex docs](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_methods_system_string.htm), the limit controls the maximum number of times that the pattern can be applied. I believe you can omit the limit altogether since the proper usage will only involve a single `:` character. That means we can simplify this to:
```
String[] recordInfo = innerMatchedText.split(':');

```